config/environments/test.rb

```diff
@@ -7,6 +7,7 @@
 
 Telegram.reset_bots
 Telegram::Bot::ClientStub.stub_all!
+Dotenv.overload('.env.local.test')
```

Comment:
In general I like the idea to have a well defined set of env variables for the tests. Perfectly valid. The thing that worries me is that this gem is 4 years not maintained and not widely used. Im wondering why it has been in the gemfile if not used before this PR..